src/routes/handlers/where-start-looking.ts

```diff
@@ -28,12 +28,12 @@ export function wherePost(req: express.Request, res: express.Response) {
         res.redirect('/in-hiding');
         break;
       } else {
-        res.send('no active countries');
+        res.redirect('/eligible-one-no-active');
```

Comment:
Might be a more meanigful slug as...

```suggestion
        res.redirect('/eligible-one-country-none-active');
```

Reply:
I've changed the routes to be more readable but I went with 'eligible-no-couontry-active'